app/react/App/Root.js

```diff
@@ -34,7 +34,17 @@ const googelFonts = (
   />
 );
 
-const headTag = (head, CSS, reduxData) => (
+const getFaviconURL = reduxData => {
+  const favicon = reduxData.settings.collection.get('favicon');
+
+  if (!favicon || favicon === '') {
+    return '/public/favicon.ico';
+  }
+
+  return favicon;
+};
+
+export const headTag = (head, CSS, reduxData) => (
```

Comment:
Dear reviewer,
I had to export this function to be able to test this feature in the more integrated possible way without spending a huge amount of time. The Root component has no tests at the moment and it requires LOTS of input to be rendered, what would have taken me forever to figure out.
If you think this is not the best approach to follow, please, let me know :)

Reply:
I agree although the Root component seems small there is some logic that should be tested in its own context.